go/ql/test/query-tests/Security/CWE-643/go.mod

```diff
@@ -1,6 +1,6 @@
 module main
 
-go 1.14
+go 1.21
```

Comment:
Is bumping the go version a concern here?

Reply:
I don't think it makes a difference. Go versions are meant to be almost entirely backwards compatible.